.github/workflows/js_wallet_publish.yml

```diff
@@ -6,39 +6,43 @@ on:
       - walletjs-*
 
 jobs:
-  initial_release:
-    name: Publish wasm package
-    runs-on: ubuntu-latest
-    outputs:
-      upload_url: ${{ steps.create_release.outputs.upload_url }}
-      version: ${{ steps.get_version.outputs.version }}
-    steps:
-      - name: Checkout code
-        uses: actions/checkout@v3
+  # initial_release:
```

Comment:
What was the motivation to comment out this job?

Reply:
I have removed it for testing purposes, forgotten to uncomment